docs/community/maintainers-guide/releasing.rst

```diff
@@ -0,0 +1,224 @@
+      is correct. Pay attention how you write, keep consistency with previous
+      sentences forms.
+4. Bump version in ``package.json`` (follow semantic versioning).
+5. Regenerate the ``package-lock.json``:
+    - ``rm package-lock.json``
+    - ``npm install``
+6. Create a release commit (message pattern:
+   ``release: v<major>.<minor>.<patch>``).
+7. Create tag and push release:
+    - ``git tag v<major>.<minor>.<patch>``
+    - ``git push inveniosoftware master v<major>.<minor>.<patch>``
+8. Regularly check ``Travis`` and ``NpmJS`` to ensure the release went through.
+
+**Backport changes to a maint release**
+
+When you need to create a new release for a maintained previous release,
+you will have to do the extra step of backporting the changes to the
+maintenance branch.
+
+After the procedure described above, you will have to:
+
+1. Checkout the ``maint`` branch and create a new release branch:
+    - ``git checkout maint-<major>.<minor>``
+    - ``git checkout -b rel-v<major>.<minor>.<patch>``
+2. Cherry-pick the commits that you need to backport, resolve any conflict.
+   (yes, cherry-pick :))
+3. Run tests.
+4. Createa new commit with updated ``changes`` file and bumped version,
+   as described above.
+5. Issue a pull request against the **maintenance branch** (
+   ``maint-<major>.<minor>``).
+6. If Travis fails:
+    - Fix issue and ensure head commit is the release commit (i.e. rebase if
+      necessary).
+7. Merge, tag and push release:
+    - ``git merge --ff-only rel-v<major>.<minor>.<patch>``
+    - ``git tag v<major>.<minor>.<patch>``
+    - ``git push inveniosoftware maint-<major>.<minor> v<major>.<minor>.<patch>``
+
+Invenio release
+---------------
+
+The pre-requirement necessary to release Invenio is that all the features in
+the various Invenio modules needed for the release have been merged and
+released. Then:
+
+1. Update the ``setup.py``:
+    - review all modules lower and upper bounds and adjust them as needed.
+2. Documentation:
+    - review documentation and make sure new features or breaking changes are
+      documented, to help users when upgrading.
+3. Prepare release notes (`see example <https://github.com/inveniosoftware/invenio/commit/f4d0aa5ac78d76228fe86754eeb3bbfe81a1854f>`_):
+    - In ``docs/releases/``, copy an existing patch-level or minor
+      release notes (e.g. ``docs/releases/v3.1.2.rst``).
+    - Edit release notes.
+    - Include the new release notes into ``docs/releases/index.rst``.
+    - Check the "Maintenance policy", e.g. is the version correct?
+      (`example <https://github.com/inveniosoftware/invenio/commit/edb863d2f5228fb158c090a69c2db7c3385b6ba3>`_).
+4. Create a release commit (message pattern:
+   ``release: v<major>.<minor>.<patch>``).
+5. Create tag and push release:
+    - ``git tag v<major>.<minor>.<patch>``
+    - ``git push inveniosoftware master v<major>.<minor>.<patch>``
+6. Regularly check ``Travis`` and ``PyPI`` to ensure the release went through.
```

Comment:
I would rephrase as in https://github.com/inveniosoftware/invenio/pull/4044/files#r474518553

Reply:
same, might need discussion IRL